gulp/browserSync.js

```diff
@@ -21,12 +23,42 @@ export function serve(done) {
     // get a fresh copy of the config
     const config = requireUncached(paths.config.themeConfig);
 
+    let serverConfig = {
+        proxy: config.dev.browserSync.proxyURL,
+        port: config.dev.browserSync.bypassPort,
+        liveReload: true,
+        https: false
+    };
+
+    // Default to included cert/key paths
+    let certPath = `${paths.config.browserSync}/wp-rig-browser-sync.crt`;
+    let keyPath = `${paths.config.browserSync}/wp-rig-browser-sync.key`;
+
+    // Use custom cert and key paths if defined
+    if( config.dev.browserSync.hasOwnProperty('certPath') ){
+        certPath = config.dev.browserSync.certPath;
+        log(colors.yellow(`Using the custom SSL certificate ${colors.bold(certPath)}`));
+    } else {
+        log(colors.yellow(`Using the default SSL certificate ${colors.bold(certPath)}`));
+    }
+    
+    if( config.dev.browserSync.hasOwnProperty('keyPath') ){
+        keyPath = config.dev.browserSync.keyPath;
+        log(colors.yellow(`Using the custom SSL key ${colors.bold(keyPath)}`));
+    } else {
+        log(colors.yellow(`Using the default SSL key ${colors.bold(keyPath)}`));
+    }
+
+    if (config.dev.browserSync.https) {
```

Comment:
We should check this logic first, because otherwise there's no point in running the rest.

Reply:
And it should all be moved inside `if (config.dev.browserSync.live)`.